spmd/tensor/api.py

```diff
@@ -72,7 +71,9 @@ def unwrap_mesh(e):
             # if this tensor is not Distributed, then return none. We will reinterpret it as replicated
```

Comment:
How about adding type annotation and fix pyre warnings when we update existing code?